.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
+}
```

Comment:
I guess this doesn't get linted... where you see these, would you mind making sure your last line in the file is empty?  (I.e., hit return.  :-) )

Reply:
👍